agent/snmpvars.hpp

```diff
@@ -36,6 +36,9 @@ enum class TruthValue : int
     False = 2,
 };
 
+#define SNMPBOOL(value)                                                        \
```

Comment:
why so many spaces before `\` ? Please put it right after the closing paren.

Reply:
It made by `clang-format` and it put white spaces again when i remove they.